### DISCUSSION TITLE
A big hearty thank you!

### DISCUSSION THREAD

**Discussion by kiramisui** [brianschrameck/arlo-cam-api#14] [Show and tell]
I got Arlo cameras years ago (7-8 I think). At the time I was quite happy with excellent quality good cameras. 
As the years passed I got more into security cameras, home assistant etc. Which is when my first frustration came with Arlo = I started testing Frigate and ofc I couldn't get RTSP stream with Arlo. But the real anger came with the increasing subscription, limitations and most of all the quality limitation they set (no subscription, no 4k recording). For me this was the single biggest "fuck you" Arlo could've done. I bought 4k cameras but then they decided to limit it. Literally cheating their customers.  
I bought a bunch of other cameras, some UNIFI, Dahua, Hikvision, tested and build myself a nice system with Frigate and some cool object detection and the HA automations I could build. 

Long story short, I placed all Arlo cameras, doorbell, solar, cables, basestations (yes plural), and wasn't getting much interest. That's when I came across Arlo-cam-api. I haven't integrated the doorbell camera yet, however all my 8 Arlo cameras are now running flawlessly on Frigate. 
I went the mediamtx route so it works with Frigate, and still have to dive deeper into how to certain camera settings like stream, audio or camera floodlights, or if that's even possible. But my 8 Arlo cameras have now:
- RTSP streams so I could include them in Frigate.
- Running with some packet losses but nothing major = video is fluent and quite perfect if I might say.
- I just got a much larger coverage as I'm using my Unifi AP directly with the Arlo VLAN instead of the stupid Arlo basestation that couldn't get very far. 
- And I literally just saved a lot of money and can finally use these cameras properly. 

Anyways, sorry for the long text: I just wanted to leave a big thank you, you sir deserve a medal!

**Comment by brianschrameck** [comment]
Glad it's working out for you! I certainly can't take credit as this is a fork from an existing project.

Are you able to get the 4K streams working? I believe they are served as HEVC on port 555?

**Reply by kiramisui** [reply]
Sorry for long time, I missed your comment.

I managed yes to get the 4k streams working. They are served on port 555 indeed.
ffprobe from rtsp stream:

  Metadata:
    title           : RTSP/RTP stream from omni Video Server
    comment         : live
  Duration: N/A, start: -2.177000, bitrate: N/A
  Stream #0:0: Audio: aac (LC), 16000 Hz, mono, fltp
  Stream #0:1: Audio: opus, 48000 Hz, stereo, fltp
  Stream #0:2: Video: hevc (Main), yuvj420p(pc, bt709), 3840x2160, 24 fps, 24 tbr, 90k tbn

I got it working pretty well for frigate with mediamtx. I am however re-encoding/transcoding the h265 stream to h264 with go2rtc as it was buggy.

Thanks again!
